localstack/services/events/events_starter.py

```diff
@@ -32,6 +33,10 @@ def send_event_to_sqs(event, arn):
     sqs_client.send_message(QueueUrl=queue_url, MessageBody=event['Detail'])
 
 
+def send_event_to_lambda(event, arn):
+    run_lambda(event=json.loads(event['Detail']), context={}, func_arn=arn)
```

Comment:
I think we should call the Lambda asynchronously here (i.e., let it run in the background). Otherwise, this call would be blocking until the Lambda has finished executing (and the `put_events` call below would fail if the Lambda fails.)
```
     run_lambda(event=json.loads(event['Detail']), context={}, func_arn=arn, asynchronous=True)
```